sdks/python/apache_beam/io/requestresponseio.py

```diff
@@ -0,0 +1,100 @@
+#
```

Comment:
rrio.py instead of the full name?

Reply:
Thank you so much for reviewing, @liferoad. I personally prefer with you the abbreviation. It was on the original Java until the Java was changed to the full `requestresponse` (without the io part of the name). To maintain consistency with the Java, I reasoned to spell out the full name. What do others prefer (cc: @Abacn , @johnjcasey , @riteshghorse , @ahmedabu98)